tests/SprykerSdkTest/Zed/AppSdk/Communication/Console/AppManifestValidateConsoleTest.php

```diff
@@ -50,13 +49,15 @@ public function testValidateManifestReturnsSuccessCodeWhenValidationIsSuccessful
     public function testValidateManifestReturnsErrorCodeAndPrintsErrorMessagesWhenValidationFailed(): void
     {
         // Arrange
-        $commandTester = $this->tester->getConsoleTester(new AppManifestValidateConsole());
+        $this->tester->haveInvalidManifestFile();
+
+        $commandTester = $this->tester->getConsoleTester(AppManifestValidateConsole::class);
 
         // Act
-        $commandTester->execute([], ['verbosity' => OutputInterface::VERBOSITY_VERBOSE]);
```

Comment:
Why did you remove it? With this, the output will not be printed to the console. And the method name is saying that you test here that it prints.